### PR TITLE
updates Signup/PostManger to have option to not log when sending to Quasar

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
- updates Signup/PostManger to have option to not log when sending to Quasar
- updates `UpdateSignupAndOrPostField` command to not log when updating Signup/Post in Quasar

#### How should this be reviewed?
👀 

#### Any background context you want to provide?
Goes along with [this discussion](https://github.com/DoSomething/rogue/pull/714#issuecomment-393259907).

#### How to deploy
https://github.com/DoSomething/rogue/blob/master/docs/development/deployments.md

### CODE REVIEW

File: app/Managers/PostManager.php [hunk 1]
```diff
@@ -72,9 +72,10 @@ public function create($data, $signupId, $authenticatedUserRole = null)
      *
      * @param \Rogue\Models\Post $post
      * @param array $data
+     * @param bool $log
      * @return \Rogue\Models\Post
      */
-    public function update($post, $data)
+    public function update($post, $data, $log)
```
Comment (sbsmith86) [inline app/Managers/PostManager.php]: I think we should give this a default value. `$log = true`

Comment (chloealee) [reply app/Managers/PostManager.php]: ah right - forgot that!

**Comment by chloealee** [comment]
added one last thing (`@TODO`) - ready for another look! :)

File: app/Console/Commands/UpdateSignupAndOrPostField.php [hunk 2]
```diff
@@ -78,7 +78,9 @@ public function handle()
                         info('rogue:updatefield: changing ' . $targetField . ' to ' . $targetNewValue . ' for signup ' . $signup->id);
                     }
 
-                    $this->signups->update($signup, [$targetField => $targetNewValue]);
+                    // Update signup but don't log that it was sent to Quasar in interest of taking up too much space in Papertrail.
+                    // @TODO: when running updates for smaller amount of records, do we want to turn logs back on?
```
Comment (sbsmith86) [inline app/Console/Commands/UpdateSignupAndOrPostField.php]: You can take this param in as a flag in the command, so you can set the option when you run the command.

Comment (sbsmith86) [reply app/Console/Commands/UpdateSignupAndOrPostField.php]: `$log = $this->option('log');`

Comment (chloealee) [reply app/Console/Commands/UpdateSignupAndOrPostField.php]: oh great call, @sbsmith86 ! Just updated to that ! :)

File: app/Console/Commands/UpdateSignupAndOrPostField.php [hunk 3]
```diff
@@ -15,7 +15,7 @@ class UpdateSignupAndOrPostField extends Command
      *
      * @var string
      */
-    protected $signature = 'rogue:updatefield {target} {targetOldValue} {targetNewValue} {--signups} {--posts} {--logfreq=10000}';
+    protected $signature = 'rogue:updatefield {target} {targetOldValue} {targetNewValue} {--signups} {--posts} {--logfreq=10000} {--log}';
```
Comment (sbsmith86) [inline app/Console/Commands/UpdateSignupAndOrPostField.php]: we should set a default for the `log` option as well. probably just have it on by default.

Comment (chloealee) [reply app/Console/Commands/UpdateSignupAndOrPostField.php]: hm @sbsmith86 I'm not sure how to set the default to `true` by default - didn't seem to work when doing `{--log=true}`, `{--log=TRUE}`, or `{--log=1}`.

Comment (sbsmith86) [reply app/Console/Commands/UpdateSignupAndOrPostField.php]: Ack maybe we don't need it since it is just an on/off flag! Sorry. this looks great.

Comment (chloealee) [reply app/Console/Commands/UpdateSignupAndOrPostField.php]: No worries - thank you for the quick review! :)

**Pull request merged**